dotnet/samples/Concepts/Search/GoogleTextSearchExample.cs

```diff
@@ -0,0 +1,170 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Text.Json;
+using Google.Apis.Http;
+using Microsoft.SemanticKernel.Plugins.Web.Google;
+using Microsoft.SemanticKernel.Search;
+
+namespace Search;
+
+/// <summary>
+/// This example shows how to create and use a <see cref="GoogleTextSearch"/>.
+/// </summary>
+public class GoogleTextSearchExample(ITestOutputHelper output) : BaseTest(output)
+{
+    /// <summary>
+    /// Show how to create a <see cref="GoogleTextSearch"/> and use it to perform a text search.
+    /// </summary>
+    [Fact]
+    public async Task UsingGoogleTextSearchAsync()
```

Comment:
Since the test is all about `Google` and examples, remove the detail overhead.

```suggestion
    public async Task TextSearchAsync()
```

Reply:
I think the extra verbosity is good in the Concept samples. A lot of the samples have pretty cryptic names which makes the content less discoverable. My thinking is that I should be able to read **just the test name** and understand what it does.

![image](https://github.com/user-attachments/assets/e6c46ae9-8805-4ac4-9353-0e8035f0e3a4)